py/pysparkling/context.py

```diff
@@ -139,12 +139,14 @@ def getOrCreate(spark, conf=None, **kwargs):
 
 
     def stop_with_jvm(self):
+        Initializer.clean_temp_dir()
```

Comment:
Update: This seems causing print of stack-traces during spark shutdown. It cannot expect that it will be fully executed. The better solution is simply skip cleanup.

Reply:
good point! Will work on this tomorrow